entity_sync.module

```diff
@@ -11,3 +11,14 @@
 function entity_sync_toolbar_alter(&$items) {
   $items['administration']['#attached']['library'][] = 'entity_sync/toolbar';
 }
+
+function entity_sync_entity_update(Drupal\Core\Entity\EntityInterface $entity) {
+  // ...
+  \Drupal::logger('app_user')->notice('Updated a user');
+  $queue = \Drupal::queue('entity_sync_export_user');
```

Comment:
Remember, we're not just going to export a user entity. So, the queue worker should be called `entity_sync_export_local_entity`.

We can do this later, but in hook_entity_insert() and hook_entity_update() we should be checking whether the entity_sync.sync.* files in the config has the entity that just got updated/created to be exported to the remote resource.